tests/plugins/inlinetoolbar/manual/skins.md

```diff
@@ -0,0 +1,6 @@
+@bender-ui: collapsed
+@bender-tags: 4.8.0, feature, inlinetoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,notification,floatingspace,inlinetoolbar,sourcearea,list,link,image,language,stylescombo
+
+Check in browser console, if there is no errors with CSS loading.
```

Comment:
> if there ~is~**are** no errors

Reply:
I have have runed this manual test.

And it **does throw** an error related to CSS loading:

```
GET http://tests.ckeditor.test:1030/apps/ckeditor/plugins/balloonpanel/skins/kama,/apps/ckeditor/skins/kama//balloonpanel.css net::ERR_ABORTED
```

Reply:
this one is fixed by #1221

Reply:
I know, but here on branch it fails. You need to write exactly what is the expected result.

Person that views it doesn't know that it's fixed by #1221.